nautobot/extras/tables.py

```diff
@@ -78,6 +79,12 @@
 {{ value.all|join:", "|truncatewords:15 }}
 """
 
+SCHEDULED_JOB_APPROVAL_QUEUE_BUTTONS = """
+<button type="button" onClick="handleDetailPostAction('{% url 'extras:scheduled_jobs_approval_request_view' scheduled_job=record.pk %}', '_dry_run')" class="btn btn-primary btn-xs"{% if not perms.extras.run_job %} disabled="disabled"{% endif %}><i class="mdi mdi-play"></i></button>
+<button type="button" onClick="handleDetailPostAction('{% url 'extras:scheduled_jobs_approval_request_view' scheduled_job=record.pk %}', '_approve')" class="btn btn-success btn-xs"{% if not perms.extras.run_job %} disabled="disabled"{% endif %}><i class="mdi mdi-check"></i></button>
+<button type="button" onClick="handleDetailPostAction('{% url 'extras:scheduled_jobs_approval_request_view' scheduled_job=record.pk %}', '_deny')" class="btn btn-danger btn-xs"{% if not perms.extras.run_job %} disabled="disabled"{% endif %}><i class="mdi mdi-close"></i></button>
```

Comment:
Possibly wrap these to improve readability?
```suggestion
<button type="button"
        onClick="handleDetailPostAction('{% url 'extras:scheduled_jobs_approval_request_view' scheduled_job=record.pk %}', '_dry_run')"
        class="btn btn-primary btn-xs"{% if not perms.extras.run_job %} disabled="disabled"{% endif %}>
    <i class="mdi mdi-play"></i>
</button>
<button type="button"
        onClick="handleDetailPostAction('{% url 'extras:scheduled_jobs_approval_request_view' scheduled_job=record.pk %}', '_approve')"
        class="btn btn-success btn-xs"{% if not perms.extras.run_job %} disabled="disabled"{% endif %}>
    <i class="mdi mdi-check"></i>
</button>
<button type="button"
        onClick="handleDetailPostAction('{% url 'extras:scheduled_jobs_approval_request_view' scheduled_job=record.pk %}', '_deny')"
        class="btn btn-danger btn-xs"{% if not perms.extras.run_job %} disabled="disabled"{% endif %}>
    <i class="mdi mdi-close"></i>
</button>
```